test/test_function_schema.py

```diff
@@ -0,0 +1,18 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
```

Comment:
if you're creating a new file - we should put it into TARGETS and some run_test.py files I think. Otherwise it won't be run in CI. Or just dump it into test_jit.py :)

Reply:
I have put it into TARGETS, forgot to add it to run_test.py. Good catch!